### ISSUE TITLE
Compilation - fails on OVL bin?

### ISSUE THREAD

**Issue by daktak** [pebble-bike/PebbleBike-PebbleWatchFace#13]
IOError: [Errno 2] No such file or directory: '/home/travis/build/daktak/JayPS-WatchFace/pebblebike/resources/data/OVL_diorite.bin'

Trying to add Travis build, how are the OVL bin files created/generated?

https://travis-ci.org/github/daktak/JayPS-WatchFace/builds/680374178

**Comment by jay3** [comment]
I think you've got to call "pebble build" twice when the repo is clean.
See wscript for details

(I don't have the sdk installed on my machine, I cannot confirm)

Jay

**Comment by daktak** [comment]
https://travis-ci.org/github/daktak/JayPS-WatchFace/builds/684600532
Tried with build twice, same error

**Comment by daktak** [comment]
https://travis-ci.org/github/daktak/JayPS-WatchFace/builds/684606373
Also tried 79 times ;-)

**Comment by jay3** [comment]
I've found time to investigate.
Something has changed since the introduction of overlays in the project and the "pebble build && pebble build" was no longer working.
Find a quick fix: https://github.com/jay3/Ventoo-PebbleWatchFace/commit/e50d4510d45a0b40e3768eb5a1d60e0b0824f5fc


Note: I don't have a travis env ready, but I've reproduced the bug in docker (https://github.com/bboehmke/docker-pebble-dev) and fixed it there. I hope it will fix it for travis.

Jay
